c/ppport.h

```diff
@@ -1171,7 +1171,7 @@ av_reify|||
 av_shift|||
 av_store|||
 av_tindex||5.017009|
-av_top_index||5.017009|
+av_len||5.017009|
```

Comment:
Was this a manual change or the result of updating this file with the latest version from CPAN?